draft-packages/tile/package.json

```diff
@@ -37,7 +37,10 @@
   },
   "dependencies": {
     "@kaizen/component-library": "^12.4.5",
-    "@kaizen/draft-button": "^5.6.5",
+    "@kaizen/button": "^1.0.4",
     "classnames": "^2.3.1"
+  },
+  "dependenciesComments": {
+    "@kaizen/draft-button": "The draft dependency should remain here until this is moved out of draft packages"
```

Comment:
```suggestion
```

_Nothing to see here (there's no draft-button)._ 👀

Reply:
This is not the draft-button you're looking for ⭐ 🧙 🤖 Consider it gone 😆